blocks/recent-posts/recent-posts.js

```diff
@@ -0,0 +1,51 @@
+import { getAllBlogs } from '../../scripts/scripts.js';
+import { createOptimizedPicture, getMetadata } from '../../scripts/lib-franklin.js';
+
+function createCard(row, style) {
+  const card = document.createElement('div');
+  if (style) card.classList.add(style);
+  if (row.image && row.title) card.prepend(createOptimizedPicture(row.image));
+  const cardContent = document.createElement('div');
+  cardContent.classList.add('card-content');
+  const link = document.createElement('a');
+  link.classList.add('blog-link');
+  link.href = row.path;
+  if (row.title) link.innerHTML = `<h3>${row.title}</h3>`;
+  cardContent.append(link);
+  card.append(cardContent);
+  return (card);
+}
+
+export default async function decorate(block) {
+  block.textContent = '';
+  let counter = 0;
+  let secondCard = 0;
+  // Make a call to get all blog details from the blog index
+  const blogList = await getAllBlogs();
+  const titleContent = getMetadata('og:title');
+  if (blogList.length) {
+    // eslint-disable-next-line
+    const sortBl = blogList.sort((objA, objB) => Number(new Date(objB.published)) - Number(new Date(objA.published)));
```

Comment:
We are not sorting here, we are leaving this up to the spreadsheet.

Reply:
Sorting is required that's why it is mentioned - Recent Posts. Otherwise it will not get the Recent posts ? @helms-charity

Reply:
fixed